ports/stm32/storage.c

```diff
@@ -64,7 +64,9 @@ void storage_init(void) {
         // Enable the flash IRQ, which is used to also call our storage IRQ handler
         // It must go at the same priority as USB (see comment in irq.h).
         NVIC_SetPriority(FLASH_IRQn, IRQ_PRI_FLASH);
+        #if (MICROPY_HW_FLASH_BACKGROUND_CACHE) || defined(MICROPY_HW_BDEV2_IOCTL)
```

Comment:
I don't think this is needed, it can enable the IRQ unconditionally.

Reply:
I figured with this enabled all the time, there's an unneccesary (albeit miniscule) cpu overhead of having the now-empty irq function running on a regular basis.

Reply:
Ah, OK.  Then maybe it's even better to just disable this line:
```
systick_enable_dispatch(SYSTICK_DISPATCH_STORAGE, storage_systick_callback);
```
Then you don't need to touch anything in flashbdev.c.

Ultimately I want to switch to a soft-timer based implementation for this storage flushing callback... but that can wait.